arch/arm/src/arm/arm_tcbinfo.c

```diff
@@ -63,6 +54,22 @@ const struct tcbinfo_s g_tcbinfo =
   TCB_REG_OFF(REG_CPSR),
 };
 
+/****************************************************************************
+ * Public Data
+ ****************************************************************************/
+
+const struct tcbinfo_s g_tcbinfo =
+{
+  TCB_PID_OFF,
+  TCB_STATE_OFF,
+  TCB_PRI_OFF,
+  TCB_NAME_OFF,
+  XCPTCONTEXT_REGS,
+  {
+    .p = g_reg_offs,
```

Comment:
uncompliant with C89, but since this is arch specific, I'm not sure if we need to care about it

Reply:
arm arch already has many similar construct, so we relax the check until we do the final decision?

Reply:
@pkarashchenko do you think it's reasonable to ignore this in the current state?

Reply:
Yes. Of course